doc/external-libs/ginkgo.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+	  "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+  <head>
+    <title>The deal.II Readme on interfacing to Ginkgo</title>
+    <link href="../screen.css" rel="StyleSheet">
+    <meta name="copyright" content="Copyright (C) 2008 - 2018 by the deal.II authors">
+    <meta name="keywords" content="deal.II">
+  </head>
+
+  <body>
+
+    <h1>Interfacing <acronym>deal.II</acronym> to Ginkgo</h1>
+
+    <p>
+        <a href="https://github.com/ginkgo-project/ginkgo"
+           target="_top">Ginkgo</a> is a
+        Numerical linear algebra software package that provides its users with
+        highly optimized fine grid level linear algebra operations. It currently has
+        the capability to solve on the CPU with the support of OpenMP and on the GPU
+        with NVIDIA's cuda libraries. For some operations it uses NVIDIA's own libraries, CuSparse
+        and CuBLAS but some routines also have self implemented cuda functions. See the documentation
+        for more details.
+    </p>
+
+    <p>
+      <acronym>deal.II</acronym> has wrapper classes to the linear algebra
+      parts of Ginkgo that provide almost the
+      same interfaces as the built-in <acronym>deal.II</acronym> linear algebra operations.
+    </p>
+
+    <h2>Installing <acronym>deal.II</acronym> with Ginkgo</h2>
+
+    <p>
+        During the cmake configuration, the following flags should be specified
+        <code>-DDEAL_II_WITH_GINKGO=ON</code>, to enable Ginkgo and the <code>-DGINKGO_DIR=DIR</code>
+        can be set with the path where Ginkgo has been installed. 
```

Comment:
I would suggest that this whole paragraph needs to be restructured slightly to be clear. May I suggest the following:

```
During the CMake configuration, the following flags should be specified:
<ul>
  <li> <code>-DDEAL_II_WITH_GINKGO=ON</code>, to enable the Ginkgo library
  <li> <code>-DGINKGO_DIR=<path_to_ginko></code>, to specify the path where Ginkgo has been installed.
</ul>
```